autogluon/utils/tabular/ml/models/lgb/callbacks.py

```diff
@@ -142,14 +148,27 @@ def _callback(env):
             _init(env)
         if not enabled[0]:
             return
-
+        if train_loss_name is not None:
+            train_loss_evals = [eval for i, eval in enumerate(env.evaluation_result_list) if eval[0] == 'train_set' and eval[1] == train_loss_name]
```

Comment:
just small comment to old code: `enumerate` is not needed.

Reply:
Thanks! Removed in latest update.